diktat-analysis.yml

```
@@ -280,4 +280,10 @@
 # Checks that floating-point values are not used in arithmetic expressions
 - name: FLOAT_IN_ACCURATE_CALCULATIONS
   enabled: true
-  configuration: {}
\ No newline at end of file
+  configuration: {}
+# Checks that function length isn't too long
+- name: TOO_LONG_FUNCTION
+  enabled: true
+  configuration:
+    maxFunctionLength: "30" # max length of function
+    isIncludeHeader: "true" # count function's header
```

```suggestion
    isIncludeHeader: "false" # count function's header
```
For our own code check let's be consistent to setup that we already have.